docs/_docs/languages/graphql.md

```diff
@@ -0,0 +1,67 @@
+---
+pageid: language-graphql
+title: GraphQL
+layout: docs
+permalink: /docs/languages/graphql/
+---
+
+Nuclide has built-in support for [GraphQL](http://graphql.org/) using the [GraphQL Language Service](https://github.com/graphql/graphql-language-service).
+
+* TOC
+{:toc}
+
+## Installing GraphQL
+
+Several [server libraries](http://graphql.org/code/) are provided for GraphQL implementation in a wide range of languages.
+
```

Comment:
Configuration setup instructions can be written here I believe. I took a pass using information from https://github.com/graphql/graphql-language-service#graphql-configuration-file-graphqlrc - please feel free to modify:

GraphQL Language Service, to provide its full feature set, will need to know some information about your GraphQL development environment. `.graphqlrc` is a GraphQL configuration file that contains this information. `.graphqlrc` can define mutliple configurations for each GraphQL environment, should you have more than one.

Make sure `.graphqlrc` file is configured and saved in your project's top level directory. For more information on how to configure, refer to [this documentation](https://github.com/graphql/graphql-language-service#graphql-configuration-file-graphqlrc).